src/tools/rustdoc-gui/tester.js

```diff
@@ -0,0 +1,83 @@
+const {Options, runTest} = require('browser-ui-test');
```

Comment:
Where does this package come from? Does it need to be installed by the user or by bootstrap?

Reply:
I said it in the PR message that it was part of the information I wanted to put inside the documentation. I also gave the command to install it (which is `npm install browser-ui-test`). ;)

Reply:
Oops, I should read :laughing: thanks!

Reply:
No, I think it'd be nice to add a comment in the code actually. I'll do that.